www/index.html

```diff
@@ -38,23 +20,154 @@
+            </div>
+            <div class="panel-body">
+                <ol>
+                    {{ #stops }}
+                        <li>{{ name }}</li>
+                    {{ /stops }}
+                </ol>
+            </div>
+        </div>
+    </script>
+
+    <script id="departures-tmpl" type="x-tmpl-mustache" >
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h3 class="panel-title">Dia de Semana</h3>
+            </div>
+            <div class="panel-body">
+                <ul class="list-inline unstyled">
+                    {{ #WEEKDAY }}
+                        <li>{{ time }}</li>
+                    {{ /WEEKDAY }}
+                </ul>
+            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h3 class="panel-title">Sábado</h3>
+            </div>
+            <div class="panel-body">
+                <ul class="list-inline unstyled">
+                    {{ #SATURDAY }}
+                        <li>{{ time }}</li>
+                    {{ /SATURDAY }}
+                </ul>
+            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h3 class="panel-title">Domingo</h3>
+            </div>
+            <div class="panel-body">
+                <ul class="list-inline unstyled">
+                    {{ #SUNDAY }}
+                        <li>{{ time }}</li>
+                    {{ /SUNDAY }}
+                </ul>
+            </div>
+        </div>
+    </script>
+
+    <script id="routes-result-tmpl" type="x-tmpl-mustache">
+        {{ #empty }}
+            <div class="alert alert-warning alert-dismissible fade in" role="alert">
+                <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+                <strong>Oops!</strong> Nenhum registro encontrado.
+            </div>
+        {{ /empty }}
+
+        {{ ^empty }}
+            <div class="table-responsive">
+                <table class="table table-hover table-striped">
+                    <thead>
+                        <tr>
+                            <th>#</th><th>Nome</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {{ #routes }}
+                        <tr data-route-id="{{ id }}" data-route-name="{{ longName }}">
+                            <td>{{ shortName }}</td>
+                            <td>{{ longName }}</td>
+                        </tr>
+                        {{ /routes }}
+                    </tbody>
+                </table>
+            </div>
+        {{ /empty }}
+    </script>
+
+
+    <script type="text/javascript" src="cordova.js"></script>
+    <!-- external dependencies -->
+    <script src="./libs/jquery/3.1.1/jquery.min.js"></script>
```

Comment:
I see several script tags here. What about a build tool, have you worked with something like that before? I mean, for javascript? Like grunt, gulp, webpack? What is the con of using the approach you used compared to a concatenated version of the files?